provider/resource_keycloak_openid_client_permission_test.go

```diff
@@ -0,0 +1,135 @@
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+)
+
+func TestAccKeycloakOpenidClientPermission_basic(t *testing.T) {
+	realmName := "tf_view-" + acctest.RandString(10)
+	clientId := "tf-" + acctest.RandString(10)
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: testAccProviderFactories,
+		PreCheck:          func() { testAccPreCheck(t) },
+		Steps: []resource.TestStep{
+			{
+				Config: testKeycloakOpenidClientPermission_basic(realmName, clientId),
+				Check:  testAccCheckKeycloakOpenidClientPermissionExists("keycloak_openid_client_permission.my_permission"),
```

Comment:
Could you add a test that checks that the policy is actually set on the permission?